workflow-pacta-cop.Rmd

```diff
@@ -0,0 +1,516 @@
+  
+  if (length(unique(peers_equity_results_user$portfolio_name)) <= number_of_investors){
+    print("More investors expected in peers_equity_results_user. The portfolio_name column should be the investor_name string provided by the user. User_id would work as an alternative.")
+  } 
+  
+  
+}else{
+  print("No Equity individual results")
+}
+
+if(file.exists(file.path(data_path, paste0(project_code, "_peers_bonds_results_portfolio_ind.rda")))){
+  peers_bonds_results_user <- read_rds(file.path(data_path, paste0(project_code, "_peers_bonds_results_portfolio_ind.rda")))
+  
+  if (length(setdiff(unique(peers_bonds_results_user$investor_name), c(peer_groups))) > 0){
+    print("Unexpected occurances in peers_bonds_results_user. The following is missing from the investor_name col:")
+    print(setdiff(unique(peers_bonds_results_user$investor_name), peer_groups))
+  } 
+  
+  if (length(unique(peers_bonds_results_user$portfolio_name)) <= number_of_investors){
+    print("More investors expected in peers_bonds_results_user. The portfolio_name column should be the investor_name string provided by the user. User_id would work as an alternative.")
+  } 
+  
+  
+}else{
+  print("No Bond individual results")
+}
+
+
+
+```
+
+Important: once the checks have been reviewed and the files look reasonable, you now need to make a new branch in pacta-data and create a Pull request with this branch so that the files can be uploaded to github from where they are accessible by the tool. Ask if you need help on this! 
+
+
+
+## Stress Test Parameters
+
+If the stress testing module is going to be included, then ensure that a parameter file with the project parameters exists as well as the transition scenario. 
+
+```{r}
+
+if(file.exists(file.path("..","StressTestingModelDev","model_parameters", paste0("model_parameters_", project_code, ".yml")))){
+  print("Parameter file exists")
+}else{print(paste0("Stress test parameter file does not exist. Go to: ", file.path("StressTestingModelDev","model_parameters"), " for a template file."))}
+
+
+if(file.exists(file.path("..","StressTestingModelDev","data-raw", "project_transition_scenarios", paste0("transition_scenario_", project_code, ".csv")))){
+  print("Transition scenario file exists")
+}else{print(paste0("Stress test parameter file does not exist. Go to: ", file.path("StressTestingModelDev","data-raw", "project_transition_scenarios"), " for a template file."))}
+
+
+```
+
+## Portfolio report
+
+Customise the portfolio report for you project. Either create new folders for the project, or use the general template. Ensure that you use the latest version of the create_interactive_report repository before creatng a new template. 
+Generally there is an option to create two outputs:
+
+- Interactive Report (required)
+
+- Executive summary (optional)
+
+
+The executive summary will not print if the files do not exist
```

Comment:
should this be restated given the merged PR from yesterday (blank PDF is dropped in place)?

Reply:
👍